gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -16,6 +16,28 @@
 
 package com.google.cloud.hadoop.fs.gcs;
 
+import static com.google.cloud.hadoop.gcsio.GoogleCloudStorageOptions.BATCH_THREADS_DEFAULT;
```

Comment:
Probably we need to converge on the default vars naming and prefix all of them with `DEFAULT_` prefix for consistency?

Reply:
All these are public constants previously defined. I will change for gRPC variables as it is not being.